lib/ops/resources/gravity/gravity.go

```diff
@@ -207,10 +210,8 @@ func (r *Resources) Create(req resources.CreateRequest) error {
 			return trace.Wrap(err)
 		}
 		r.Printf("Updated monitoring alert target %q\n", target.GetName())
-	case "":
-		return trace.BadParameter("missing resource kind")
 	default:
-		return trace.BadParameter("unsupported resource %q, supported are: %v",
+		return trace.NotImplemented("unsupported resource %q, supported are: %v",
```

Comment:
Nit: Not sure if "not implemented" is semantically better suited here than "bad parameter" tbh, as it usually implies that something is not implemented _yet_ while this is supposed to filter out garbage/typos.